### PR TITLE
OCPBUGS#33115: Update backup/restore, disaster recovery for HCP

### PR DESCRIPTION
**Pull request by kowen-rh**
Version(s):
4.15+

Issue:
[OCPBUGS-33115](https://issues.redhat.com/browse/OCPBUGS-33115)

Link to docs preview:
[Recovering an unhealthy etcd cluster](https://82468--ocpdocs-pr.netlify.app/openshift-enterprise/latest/hosted_control_planes/hcp_high_availability/hcp-recovering-etcd-cluster.html) 

QE review:
- [x] QE has approved this change.

Additional information:
N/A

### CODE REVIEW

File: modules/hosted-cluster-etcd-status.adoc [hunk 1]
```diff
@@ -14,19 +14,24 @@ You can check the status of the etcd cluster health by logging into any etcd pod
 +
 [source,terminal]
 ----
-$ oc rsh -n <hosted_control_plane_namespace> -c etcd <etcd_pod_name>
+$ oc rsh -n openshift-etcd etcd <etcd_pod_name>
```
Comment (mburke5678) [inline modules/hosted-cluster-etcd-status.adoc:17]: Don't we need the `-c` here, assuming the `etcd` is a container?

**Comment by mburke5678** [comment]
@kowen-rh Can I ask one question ^^ before we merge?

File: modules/hosted-cluster-etcd-status.adoc (hunk 1 above)
Comment (kowen-rh) [reply modules/hosted-cluster-etcd-status.adoc]: You're absolutely correct, thank you! Updated now. :slightly_smiling_face:

**Pull request merged**